src/test/BarcodeSearchController.test.ts

```diff
@@ -0,0 +1,40 @@
+import * as chai from 'chai';
+import * as sinon from 'sinon';
+import BarcodeSearchController from '../controllers/BarcodeSearchController';
+import ChipsDao from '../daos/CHIPS/ChipsDao';
+chai.use(require('sinon-chai'));
+
+describe('barcode search controller', ()=>{
+
+    const req = {
+        query: {
+            search: 'barcode'
+        }
+    }
+    const res = {
+        render: sinon.spy()
+    };
+
+    before(()=>{
+        const chipsDao = {
+            makeQuery: function() { }
+        };
+        const daoStub = sinon.stub().callsFake(()=>{return chipsDao;});
+        sinon.stub(chipsDao, 'makeQuery').resolves({rows: ['data'] });
+        Object.setPrototypeOf(ChipsDao, daoStub);
+    })
+    it('test getSearchPage calls response render', ()=>{
```

Comment:
I think it may look a little easier on the eye if there was a new line before every test.